src/OrchardCore.Build/Dependencies.AspNetCore.props

```diff
@@ -24,16 +24,6 @@
     <MicrosoftExtensionsPackagesVersion>8.0.0</MicrosoftExtensionsPackagesVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$(TargetFramework) == 'net7.0'">
```

Comment:
WE need to keep it commented as an example, otherwise we'll forget how to do that when net9.0 ships. We had this problem last time and the templates got broken.

Reply:
Re-added it with a comment.